app/controllers/internal/broadcasts_controller.rb

```diff
@@ -24,6 +24,15 @@ def edit
 
     def create
       @broadcast = Broadcast.new(broadcast_params)
+      broadcastable_type = broadcast_params[:broadcastable_type]
+
+      unless Broadcast::ALLOWED_TYPES.include?(broadcastable_type)
```

Comment:
We have [this broadcastable_type validation in the model](https://github.com/forem/forem/pull/9410/files#diff-5a92924aeb092d84d6360b99c8e6ad1aR11) and in addition, we have what seems to be a similar manual validation here in the controller twice. Could we remove this and just rely on the model validation?

Reply:
This was more of just an extra safety precaution, so it _should_ be fine (🤞 ) to remove this validation from both the #create and #update methods!